bin/runtime/src/lib.rs

```diff
@@ -210,7 +208,12 @@ impl frame_system::Config for Runtime {
 
 impl pallet_randomness_collective_flip::Config for Runtime {}
 
+parameter_types! {
+    pub const MaxAuthorities: u32 = 100;
```

Comment:
have we decided on that? or just a reasonable arbitrary number collapsing with `MaxSignatories` from multisig?

Reply:
I think it is copied from Substrate full node, not sure what should be a correct way to decide what value should take such variable

Reply:
It would be nice to see some code-comment for it then.

Reply:
this was POOA (pulled out of ass). I raised this number to follow polkadot, comment added as per suggestion.